Home.html

```diff
@@ -14,64 +14,99 @@
+                            <h6>Visa</h6>
+                            <p>Visa application services for more than 10 countries</p>
+                    </div>
+                    <div id="main-program-divs" class=" d-flex ps-3 pe-3 ms-3 me-3 mb-2">
+                        <img src="assets/tickets.png" alt="Tickets">
+                        <h6>Tickets</h6>
+                        <p>We take care of the tickets so you don't have to worry about a thing</p>
+                    </div>
+                    <div id="main-program-divs" class=" d-flex ps-3 pe-3 ms-3 me-3 mb-2">
+                        <img src="assets/bank.png" alt="Bank">
+                        <h6>Bank</h6>
+                        <p>Don't have a bank statement, no problem we will take care of that</p>
+                    </div>
+                    <div id="main-program-divs" class=" d-flex ps-3 pe-3 ms-3 me-3 mb-2">
+                        <img src="assets/patient.png" alt="passport">
+                        <h6>Medical</h6>
+                        <p>Including all the medical files you need to apply for a medical visa</p>
+                    </div>
                 </div>
                 <div class="text-center">
-                    <button class="orange-button">Join Afghan Travel Agency</button>
+                    <button class="orange-button">SEE THE WHOLE PROGRAM</button>
                 </div>
             </section>
-            <section class="featured-speakers mt-4 mb-4">
-                <h5 class="text-center">Featured Speakers</h5>
+            <section class="mt-4 mb-4">
+                <h5 class="featured-speakers-hx text-center">Featured Speakers</h5>
                 <div class="hr-orange"></div>
+                <div id="featured-speakers">
+
+                </div>
+            </section>
+            <button class="more mb-4 mt-4"><span class="more-text">MORE</span><img class="arrow" src="assets/arrow-down-sign-to-navigate.png" alt="Arrow down"></button>
+            <section class="text-center partners partner pb-4 text-white">
+                <h4 class="pt-4">Partner</h4>
+                <div class="hr"></div>
+                <div class="text-secondary">
+                    <div class="row-1 d-flex justify-content-center">
+                        <img src="assets/firefox.png" alt="mozilla">
+                        <img src="assets/google.png" alt="google">
+                        <img src="assets/airbnb.png" alt="airbnb">
+                        <img src="assets/meta.png" alt="meta">
+                        <img src="assets/tesla.png" alt="tesla">
+                    </div>
+                </div>
             </section>
-            <button class="more pt-2 pb-2 mb-4 mt-4"><span class="more-text">MORE</span><img class="arrow" src="assets/arrow-down-sign-to-navigate.png" alt="Arrow down"></button>
+            <div id="hidden" class="footer-div pt-3 pb-3 ps-4 pe-4 d-flex align-items-center">
```

Comment:
[OPTIONAL] I think it would be better to use semantic footer tag instead of div. So, your webpage could be more SEO friendly.